### ISSUE TITLE
Inaccurate contrast ratio

### ISSUE THREAD

**Issue by tolokoban** [enyancc/vscode-ext-color-highlight#111]
This extension is fantastic and I love it!

Here is a suggestion to improve contrasts for people with bad eyes (like mine).

With my configuration, it highlights the background with the appropriate color.
But I notices that for some colors, according to WCAG 2.0, the text should be black instead of white or vice-versa.

Here are some examples where the color are reversed. The first column is the background color, and the other columns are the contrast ratio for black and white. A good contrast ratio is over __4.5__.

Background | `#000000` | `#ffffff`
-----------------|---------|---------
`#5871ff` | __5.22__ | 4.02
`#b00020` | 2.86 | __7.32__

You can check the ratios on [https://contrast-ratio.com](https://contrast-ratio.com)

Here are the formulas to compute:
* [Contrast ratio](http://www.w3.org/TR/2008/REC-WCAG20-20081211/#contrast-ratiodef)
* [Luminance](https://www.w3.org/TR/2008/REC-WCAG20-20081211/#relativeluminancedef)
